README.md

```diff
@@ -1,186 +1,246 @@
+|     ![GitHub logo](https://simpleicons.org/icons/github.svg)      |    GitHub    | ``python -m pip install https://github.com/poliastro/poliastro/archive/main.zip`` |
 
-# Problems and suggestions
+For other installation methods, see the [alternative installation methods].
 
-If for any reason you get an unexpected error message or an incorrect
-result, or you want to let the developers know about your use case,
-please open a new issue in the [issue
-tracker](https://github.com/poliastro/poliastro/issues) and we will try
-to answer promptly.
 
-# Contributing
+## Documentation
 
-poliastro is a community project, hence all contributions are more than
-welcome! For more information, head to
-[CONTRIBUTING.md](https://github.com/poliastro/poliastro/blob/main/CONTRIBUTING.md).
+Complete documentation, including a [quickstart guide] and an [API reference], can
+be read on the wonderful [Read the Docs]. Multi-version documentation includes:
 
-# Support
+* [Development documentation](https://docs.poliastro.space/en/latest/)
+* [Stable documentation](https://docs.poliastro.space/en/stable/)
 
-[![mailing](https://img.shields.io/badge/mailing%20list-groups.io-8cbcd1.svg?style=flat-square)](https://groups.io/g/poliastro-dev)
-[![Join the chat at http://chat.poliastro.space/](https://img.shields.io/matrix/poliastro:matrix.org.svg?style=flat-square)](http://chat.poliastro.space/)
 
-Release announcements and general discussion take place on our [Mailing
-List](https://groups.io/g/poliastro-dev) .
+## Examples, background and talks
 
-For further clarifications and discussions, feel free to join Poliastro
-[Chat Room](http://chat.poliastro.space/).
+There is a great variety of examples demostrating the capabilities of
+poliastro. Examples can be accessed in various ways:
 
-# Citing
+* Examples source code collected in the [examples directory]
+* Rendered [gallery of examples] presented in the documentation
+* Interactive examples powered by [binder] so users can try poliastro without installing it
 
-If you use poliastro on your project, please [drop me a
-line](mailto:hello@juanlu.space).
+poliastro is also promoted through conferences and talks. These are the latest
+talks in some of the most popular conferences about scientific software:
 
-You can also use the DOI to cite it in your publications. This is the
-latest one:
+| **Conference** |                                                   **Talk**                                                   |
+|:--------------:|:------------------------------------------------------------------------------------------------------------:|
+|   SciPy 2022   | [Per Python ad astra: Interactive Astrodynamics with poliastro](https://www.youtube.com/watch?v=0GqdIRdDe9c) |
+|   OSCW  2019   |        [Interplanetary mission analysis with poliastro](https://www.youtube.com/watch?v=0GqdIRdDe9c)         |
```

Comment:
You could add:

https://pyvideo.org/europython-2016/per-python-ad-astra.html

https://pyvideo.org/euroscipy-2019/can-we-make-python-fast-without-sacrificing-readability-numba-for-astrodynamics.html

Reply:
Sounds good!